Index: xutils-dev-7.7.2/xorg-cf-files/linux.cf
===================================================================
--- xutils-dev-7.7.2.orig/xorg-cf-files/linux.cf
+++ xutils-dev-7.7.2/xorg-cf-files/linux.cf
@@ -86,6 +86,19 @@ XCOMM platform:  $XFree86: xc/config/cf/
 # define BuildPDFdocs NO
 #endif
 
+#ifndef ProjectRoot
+# define ProjectRoot   /usr
+#endif
+#ifndef ManDirectoryRoot
+# define ManDirectoryRoot      /usr/share/man
+#endif
+#ifndef AlternateUsrLibDir
+# define AlternateUsrLibDir NO
+#endif
+#ifndef AlternateIncRoot
+# define AlternateIncRoot NO
+#endif
+
 #ifndef LinuxBinUtilsMajorVersion
 # define LinuxBinUtilsMajorVersion	DefaultLinuxBinUtilsMajorVersion
 #endif
