Index: xutils-dev-7.7.2/xorg-cf-files/X11.tmpl
===================================================================
--- xutils-dev-7.7.2.orig/xorg-cf-files/X11.tmpl
+++ xutils-dev-7.7.2/xorg-cf-files/X11.tmpl
@@ -1512,10 +1512,10 @@ FCHOWN_DEFINES = -DHAS_FCHOWN
 #define DocPdfDir $(DOCDIR)/PDF
 #endif
 #ifndef FontDir
-#define FontDir $(LIBDIR)/fonts
+#define FontDir /usr/share/fonts/X11
 #endif
 #ifndef FontEncDir
-#define FontEncDir $(LIBDIR)/fonts/encodings
+#define FontEncDir /usr/share/fonts/X11/encodings
 #endif
 #ifndef AdmDir
 #define AdmDir /usr/adm
